### ISSUE TITLE
"Estimated Play Time" form takes up a lot of space in the page

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb#1106]
> It takes too much space on the page.

_Originally posted by @salty-horse in https://github.com/iftechfoundation/ifdb/issues/1056#issuecomment-2466014193_

**Comment by dfabulich** [comment]
I can imagine a few approaches here.

1. If we add it to the `/review` page #1052, maybe we could hide it from the side bar…?
2. Or we link to a separate form, e.g. "Vote on how long it takes to finish this game" (this would eliminate the need to update it live #1053)
3. Or it could be a collapsed section by default; if you click on "Vote on how long it takes to finish this game," the form would expand on the spot.

**Comment by salty-horse** [comment]
It belongs in the sidebar, but could be hidden you mark the game as played.

**Comment by dfabulich** [comment]
Maybe it would be collapsed until you mark the game as played?

**Comment by dfabulich** [comment]
At least [alyshkalia](https://intfiction.org/t/ifdb-field-for-length-of-time-update-this-feature-is-now-live/72064/58?u=dfabulich) thinks it's fine. Maybe it's fine?

**Comment by salty-horse** [comment]
Why allow users to interact with the time controls if they just give an error when "I played it" is unmarked?

**Comment by alice-blue** [comment]
The "I've played it" checkbox is not very prominent, especially compared to the star rating controls, and people may not think to check it, or they might not think that checking it will actually do anything significant.

If we were to hide the *rating stars* until people checked "I've played it," people might notice, because people know there are supposed to be rating controls. (But they still might not know why the rating controls disappeared. We might need an explanation there, like "To rate this game or vote on its estimated play time, check "I've played it.")

But the estimated time controls are so new that if only those were hidden, many people might not realize they existed.

One possibility might be to have both the star rating controls and the time vote controls visible in collapsed form until people check "I've played it."

Although if we did that, we'd want to figure out what to do if people entered a rating or a time vote, and then *unchecked* the "I've played it" box. Would we want to collapse those controls, or should they stay open to show that the person does have a rating / time vote submitted?

**Comment by salty-horse** [comment]
The problem you're describing makes sense. I have no idea how many users do or don't use the "I've played it".

There was a discussion about marking "I played it" automatically when you rate a game. Maybe setting a time should also automatically mark "I played it"...?

In the forums, people didn't complain about the Estimated Play Time form, so maybe it's just me, and not something to worry about.
